Cargo.toml

```diff
@@ -9,3 +9,6 @@ members = [
   "dsl_compile"
 , "fields", "recursion"]
 resolver = "2"
+[patch."https://github.com/privacy-scaling-explorations/halo2"]
+
+halo2_proofs = { git = "https://github.com/georgwiese/halo2", branch = "make-emit-public-v0.3.0", features = ["circuit-params"] }
```

Comment:
Unfortunately we need this because PSE Halo2's main branch has a lot of breaking changes after the 0.3.0 release which we haven't updated yet.

Reply:
actually I think you guys are not interested in Halo2, right? So this can be removed

Reply:
yeah, I think eStark is more suitable for us.